pkg/workflows/wasm/wasm.go

```diff
@@ -72,13 +79,13 @@ func newModule(binary []byte) (*module, error) {
 		"sendResponse",
 		func(caller *wasmtime.Caller, ptr int32, ptrlen int32) {
 			b, innerErr := safeMem(caller, unsafe.Pointer(uintptr(ptr)), ptrlen)
-			if err != nil {
+			if innerErr != nil {
 				outerErr = innerErr
 				return
 			}
 
 			innerErr = proto.Unmarshal(b, &resp)
-			if err != nil {
+			if innerErr != nil {
```

Comment:
@cedric-cordenier modifying these two, I think this is the right error, but please check it

Reply:
Good catch :) Thank you!